pages/agent/v3/tokens.md

```diff
@@ -51,44 +53,190 @@ To [create an agent token](/docs/apis/rest-api/clusters#agent-tokens-create-a-to
 curl -H "Authorization: Bearer $TOKEN" \
   -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/clusters/{cluster.id}/tokens" \
   -H "Content-Type: application/json" \
-  -d '{ "description": "A description" }'
+  -d '{
+    "description": "A description",
+    "allowed_ip_addresses": "0.0.0.0/0"
+  }'
 ```
 
 where:
 
-- `$TOKEN` is an [API access token](https://buildkite.com/user/api-access-tokens) scoped to the relevant _Organization_ and _REST API Scopes_ that your agent needs access to in Buildkite.
+<%= render_markdown partial: 'apis/descriptions/rest_access_token' %>
 
-- `{org.slug}` can be obtained:
+<%= render_markdown partial: 'apis/descriptions/rest_org_slug' %>
 
-    * From the end of your Buildkite URL after accessing the _Pipelines_ page of your organization in Buildkite.
+<%= render_markdown partial: 'apis/descriptions/rest_cluster_id' %>
 
-    * By running the [List organizations](/docs/apis/rest-api/organizations#list-organizations) REST API query to obtain this value from `slug` in the response. For example:
+<!--alex ignore clearly-->
 
-        ```curl
-        curl -H "Authorization: Bearer $TOKEN" "https://api.buildkite.com/v2/organizations"
-        ```
+- <%= render_markdown partial: 'apis/descriptions/common_agent_token_description_required' %>
 
-- `{cluster.id}` can be obtained:
+- <%= render_markdown partial: 'apis/descriptions/rest_allowed_ip_addresses' %>
```

Comment:
FYI, there is a good reason as to why some content reuse snippets (partials) include the initial bullet point and others don't. It has to do with the need to include additional content after the point at which the partial is used in the content and have it formatted correctly.